domains/alexjoseph.json

```
@@ -0,0 +1,12 @@
+{
+  "description": "This is my first githib page",
+  "repo": "https://github.com/alec1563/alexjoseph.github.io",
+  "owner": {
+    "username": "alec1563",
+    "email": "aj71563@gmail.com"
+  },
+  "record": {
+    "CNAME": "https://alec1563.github.io/alexjoseph.github.io/",
```

CNAME has to be a host

```suggestion
    "CNAME": "alexjoseph.github.io",
```